pkg/gofr/gofr.go

```diff
@@ -244,6 +254,17 @@ func (a *App) Shutdown(ctx context.Context) error {
 	return err
 }
 
+func (*App) isPortAvailable(port int) bool {
+	conn, err := net.DialTimeout("tcp", fmt.Sprintf(":%d", port), checkPortTimeout)
+	if err != nil {
+		return true
+	}
+
+	conn.Close()
+
+	return false
+}
```

Comment:
This method do not use App. It could be a simple function.

But it could also be moved to `internal/port.go` that was just added with #1317 by @Umang01-hash

Reply:
Did you forget to push?

I don't see the changes, but this was resolved